processing/src/main/java/org/apache/druid/frame/read/columnar/StringFrameColumnReader.java

```diff
@@ -214,7 +192,7 @@ private static long getStartOfStringDataSection(
   }
 
   @VisibleForTesting
```

Comment:
is `@VisibleForTesting` still true if this is private now?

Reply:
removed